packages/contentstack-auth/src/commands/auth/tokens/index.ts

```diff
@@ -1,14 +1,13 @@
 import { Command } from '@contentstack/cli-command';
-import cli from 'cli-ux';
-import { logger, cliux, configHandler } from '@contentstack/cli-utilities';
+import { logger, cliux, ux, configHandler } from '@contentstack/cli-utilities';
 
 export default class TokensListCommand extends Command {
   private readonly parse: Function;
   static run;
   static description = 'Lists all existing tokens added to the session';
   static aliases = ['tokens'];
   static examples = ['$ csdx auth:tokens'];
-  static flags = cli.table.flags(); // use the cli table flags as it displays tokens in table
+  static flags = ux.table.flags(); // use the cli table flags as it displays tokens in table
```

Comment:
do we need separate interface for table flags, cant we merge it with cliux

Reply:
Add computer property in CLIInterface and reading from there.